starter-code/index.html

```diff
@@ -1,31 +1,67 @@
 <!DOCTYPE html>
 <html lang="en">
 <head>
+    <meta charset="UTF-8">
+		<meta name="viewport" content="width=device-width, initial-scale=1.0">
+		<meta http-equiv="X-UA-Compatible" content="ie=edge">
+		<link rel="stylesheet" href="css/main.css">
   <title>Instagram</title>
 </head>
 <body>
-  Instagram
+    <div id="main">
+      <div class="center-box">
+          <img class="phones" src="images/phones.png">
+          <div class="leftBox">
+             <div class="leftBox-Top">
+                <img class"logo" src="images/brand.png">
+                <div class="button">
+                  <img class="home" src="images/home.png">
+                  <span class="button-text">Log in</span>
+                </div>
+            </div>
+            <div class="leftBox-Bottom">
```

Comment:
Super confusing naming... centerBox, leftBox-Bottom... (Actually, you see the left box on the right of the screen).

Try to avoid naming classes with their position/color...

Try to avoid camelCase to name classes.